src/azure-cli/azure/cli/command_modules/acs/_validators.py

```diff
@@ -197,6 +197,22 @@ def validate_load_balancer_outbound_ip_prefixes(namespace):
             raise CLIError("--load-balancer-outbound-ip-prefixes cannot contain whitespace")
 
 
+def validate_load_balancer_outbound_ports(namespace):
```

Comment:
total ports allocated to VMs cannot exceed 64k * number_of_front_end_ip. I wonder if we should add this check

Reply:
Let's do it - we have it in RP but probably makes sense to have it here in the client too. Will add

Reply:
added